cod.md

```diff
@@ -25,23 +26,109 @@ No subtle -isms. Our last social rule bans subtle racism, sexism, homophobia, tr
 
 If you see a subtle -ism at OSCamp, you can point it out to the relevant person, either publicly or privately, or you can ask one of the organizers to say something. After this, we ask that all further discussion move off of public channels. If you are a third party, and you don't see what could be biased about the comment that was made, feel free to talk to the organizers. Please don't say, "Comment X wasn't homophobic!" Similarly, please don't pile on to someone who made a mistake. The "subtle" in "subtle -isms" means that it's probably not obvious to everyone right away what was wrong with the comment.
+<br>
 
-####Harassment
+##### Harassment
 
 OSCamp does not tolerate harassment in any form.
 
 Harassment includes (but is not limited to) offensive comments related to gender, gender identity, gender presentation, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, nationality, or religion; sexual images in public spaces; deliberate intimidation, stalking, or following; harassing photography or recording; sustained disruption of talks or other events, inappropriate physical contact or unwelcome sexual attention.
+<br>
 
-####Reporting
+
+##### Reporting
 
 If you are being harassed in a OSCamp space, notice that someone else is being harassed, or have any other concerns, please announce the organizers immediately (either contact one of us directly, or send us an email at contact@oscamp.eu). You do *not* need to be the target of harassment to speak up. Making OSCamp the best possible community is everyone's responsibility.
 
 If an incident is reported during an event, the organizers will be happy to help participants contact venue security or local law enforcement, provide escorts, or otherwise assist those experiencing harassment to feel safe for the duration of the event.
 
 We will respect confidentiality requests for the purpose of protecting victims of abuse. At our discretion, we may publicly name a person about whom we’ve received harassment complaints, or privately warn third parties about them, if we believe that doing so will increase the safety of OSCamp members or the general public. We will not name harassment victims without their affirmative consent.
+<br>
+
 
-####Consequences
+##### Consequences
 
 Participants asked to stop any harassing behavior are expected to comply immediately.
 
 If a participant engages in harassing behavior, the organizers may take any action they deem appropriate, up to and including expulsion from all OSCamp spaces and events and identification of the participant as a harasser to other OSCamp members or the general public.
+<br>
+
+
+##### Hacker Manifesto
```

Comment:
As adauga ceva introductiv pentru manifest: "We encourage goodwill and a prosperous atitude towards learning. We encourage you to be different. Or be like everybody else." (ultimele doua propozitii in spiritul manifestului)